octoprint_v8theme/static/js/v8theme.js

```diff
@@ -9,6 +9,8 @@ $(function() {
             $("#control_link").addClass("active");
             $("#control").prepend($("#temp").contents());
             $("#control").addClass("active");
+            $("#temperature-graph").closest(".row").removeAttr("style");
+            $("#temperature-graph").closest(".row").attr("class", "row-fluid");
```

Comment:
The benefit is that you don't need to grab the current value and do string concatenation if you're adding a class.  Ditto for removing.  But in this case, you're setting it to a single class so whatever.  I just figured you didn't know about `addClass()`